packages/paste-website/package.json

```diff
@@ -65,10 +65,12 @@
     "gatsby": "^2.19.19",
     "gatsby-image": "^2.2.41",
     "gatsby-plugin-algolia": "^0.11.0",
```

Comment:
Should we remove this now?

Reply:
Yup. Removed all the old search deps and files.